### ISSUE TITLE
Expose the SQL CLI in the Astro CLI

### ISSUE THREAD

**Issue by tatiana** [astronomer/astro-sdk#929]
**Please describe the feature you'd like to see**

One of the requirements for 0.1 is that it should be integrated into the [Astro CLI](https://github.com/astronomer/astro-cli).

Depends on: #828 (analysis of the method and conclusion)
Depends on: #922  (spike on the candidate approaches)

**Describe the solution you'd like**
By the end of this ticket, we should be able to run the SQL CLI features from within the Astro CLI, using the approach decided on #828 

**Acceptance Criteria**

- [ ] All checks and tests in the CI should pass
- [ ] Unit tests (90% code coverage or more, [once available](https://github.com/astronomer/astro-sdk/issues/191))
- [ ] Integration tests (if the feature relates to a new database or external service)
- [ ] Docstrings in [reStructuredText](https://peps.python.org/pep-0287/) for each of the methods, classes, functions and module-level attributes (including Example DAG on how it should be used)
- [ ] Exception handling in case of errors
- [ ] Logging (are we exposing useful information to the user? e.g. source and destination)
- [ ] Improve the documentation (README, Sphinx, and any other relevant)
- [ ] How to use Guide for the feature ([example](https://airflow.apache.org/docs/apache-airflow-providers-postgres/stable/operators/postgres_operator_howto_guide.html))

**Comment by tatiana** [comment]
This involves:
1. Having the flag for installation
2. Having the subcommands

**Comment by pankajkoti** [comment]
Decide on how to maintain the docker image with `astro-sql-cli` pypi dependency installed.

**Comment by tatiana** [comment]
We have an alpha release of the Astro CLI (1.7) with the SQL CLI:
https://github.com/astronomer/astro-cli/releases/tag/v1.7.0

To install:
```
sudo bash < <(curl -sSL https://install.astronomer.io/) -s v1.7.0
```

**Comment by tatiana** [comment]
The pending work ATM is to increase the test coverage, so we are allowed to be part of the public 1.7.0 release of the Astro CLI.

**Comment by pankajkoti** [comment]
Yes, @tatiana The PR is merged. Created a new ticket for enhancing coverage: https://app.zenhub.com/workspaces/sql-cli-631f43da09a3a67f2c795913/issues/astronomer/astro-sdk/1153
